buildspec.json

```diff
@@ -1,88 +1,67 @@
 {
     "dependencies": {
         "obs-studio": {
-            "version": "28.0.1",
-            "repository": "https://github.com/obsproject/obs-studio.git",
-            "branch": "master",
-            "hash": "e8dc70d0eef4503378d6ac300e680215eb5c9379"
-        },
-        "prebuilt": {
-            "version": "2022-08-02",
-            "baseUrl": "https://github.com/obsproject/obs-deps/releases/download",
-            "label": "Pre-built obs-deps",
+            "version": "29.1.0-beta4",
+            "baseUrl": "https://github.com/obsproject/obs-studio/archive/refs/tags",
+            "label": "OBS sources",
             "hashes": {
-                "macos-x86_64": "7637e52305e6fc53014b5aabd583f1a4490b1d97450420e977cae9a336a29525",
-                "macos-arm64": "755e0fa69b17a3ae444e1befa9d91d77e3cafe628fbd1c6333686091826595cd",
-                "macos-universal": "de057e73e6fe0825664c258ca2dd6798c41ae580bf4d896e1647676a4941934a",
-                "windows-x64": "2192d8ce780c4281b807cd457994963669e5202659ecd92f19b54c3e7d0c1915",
-                "windows-x86": "9f8582ab5891b000869d6484ea591add9fbac9f1c91b56c7b85fdfd56a261c1b"
+                "macos": "61ff20291c3f1e24ad7dfbc8c7078258a0141b15e749d7020143fa572da17d63",
+                "windows-x64": "93e3055358198c54c1d87e3c6a1600fed2b349d41335faf016e15473da0c8ce9"
```

Comment:
Can probably change this to 29.1.0 or 29.1.1. I do wonder if it would be "less disruptive" to simply change the versions and hashes for buildspec in a separate small PR and keep changes in this PR to ones necessary for the broader updates.

Reply:
Yeah I updated this to 29.1.1.